sentry_sdk/profiler.py

```diff
@@ -352,13 +374,25 @@ def _should_profile(hub):
     return random.random() < float(profiles_sample_rate)
 
 
-@contextmanager
-def start_profiling(transaction, hub=None):
-    # type: (sentry_sdk.tracing.Transaction, Optional[sentry_sdk.Hub]) -> Generator[None, None, None]
+if PY33:
 
-    # if profiling was not enabled, this should be a noop
-    if _should_profile(hub):
-        with Profile(transaction, hub=hub):
+    @contextmanager
+    def start_profiling(transaction, hub=None):
+        # type: (sentry_sdk.tracing.Transaction, Optional[sentry_sdk.Hub]) -> Generator[None, None, None]
+
+        # if profiling was not enabled, this should be a noop
+        if _should_profile(hub):
+            with Profile(transaction, hub=hub):
+                yield
+        else:
             yield
-    else:
+
+else:
+    import warnings
+
+    warnings.warn("profiling is only supported on Python >= 3.3")
```

Comment:
I think this warning needs to be in the `setup` function instead so it doesn't appear from just importing this module